packages/@uppy/dashboard/src/Dashboard.jsx

```diff
@@ -53,7 +53,7 @@ export default class Dashboard extends UIPlugin {
 
     this.defaultLocale = locale
 
-    // set default options
+    // set default options, must be kept in sync with packages/@uppy/react/src/DashboardModal.js
```

Comment:
As we import Dashboard anyway, couldn't we export and use the default options?

Reply:
No, the linter wants them in the same file :/

Reply:
The linter is here to serve us, not the other way around. If we feel it's more maintainable to not having to remember to keep options in sync across the repo — we should slap an `eslint-disable` on it and reuse the settings :)

Reply:
If we want to ignore the warning, we don’t need default props at all (I think) we can simply go back to how it was before that and pass the props along using `…this.props`. There could be good reasons not to do that though, I suppose the lint rule is there for a reason.

Reply:
I read it wrong, I thought we had the same `defaultOptions` object in both Dashboard and react/Dashboard. But the latter has prop types instead of the same object. Probably makes most sense yeah